viewing_party/party.py

```diff
@@ -1,23 +1,192 @@
 # ------------- WAVE 1 --------------------
+def create_movie(title: str, genre: str, rating: float):
+    """ Create a movie dictionary utilizing the arguments and return it.
+    If one of the arguments is Falsy, instead return None.
 
-def create_movie(title, genre, rating):
-    pass
+    Keyword arguments:
+    title -- a string movie title
+    genre -- a string genre name
+    rating -- a float value
+    """
+    if title and genre and rating:

```

Comment:
Great job with this method! Couple of things to note here.

You could convey intent a little more explicitly by pulling "return None" into it's own guard clause with (above line 11 so it's the first line of code that executes in this method):
```python
if not title or not genre or not rating:
      return None
```
then proceed to create your movie dict

This way, when another developer comes along, I can immediately recognize "Oh if these values aren't valid, then we'll return None. The way your code is currently structured, I can tell at some point this method will return None, but it's not as immediately clear.

Also a note about guard clauses - The Guard Clause pattern helps in these cases by “inverting” that way of thinking and rapidly returning false conditions and keeping the important code blocks (usually the longest) in the lowest indentation level possible. You can also create your dictionary with literal syntax and return it without defining it as a variable first (on line 12)
```python
return {
        "title": title,
        "genre": genre,
        "rating": rating        
    }
```